demo-shell/karma.conf.js

```diff
@@ -50,7 +50,7 @@ module.exports = function (config) {
         colors: true,
         logLevel: config.LOG_INFO,
         autoWatch: true,
-        browsers: ['Chrome'],
+        browsers: [/*'Chrome',*/ 'ChromeHeadless'],
```

Comment:
please do not leave code comment out just remove it

Reply:
Removed comments